Description: add support for command line argument as initial content
Author: Fernando Vezzosi <fv@linuxvar.it>

--- a/src/main.cc
+++ b/src/main.cc
@@ -619,6 +619,10 @@
     gtk_completion_line_last_history_item(GTK_COMPLETION_LINE(compline));
   }
 
+  if(argc == 2 && argv[1]){
+    gtk_entry_set_text(GTK_ENTRY(compline), argv[1]);
+  }
+
   gtk_box_pack_start(GTK_BOX(hbox), compline, TRUE, TRUE, 0);
   
   int prefs_top = 80;
